src/Surfnet/StepupMiddleware/CommandHandlingBundle/Tests/Configuration/CommandHandler/InstitutionConfigurationCommandHandlerTest.php

```diff
@@ -47,14 +48,35 @@ class InstitutionConfigurationCommandHandlerTest extends CommandHandlerTest
      * @test
      * @group command-handler
      */
-    public function an_ra_location_can_be_added_to_an_existing_institution_configuration()
+    public function an_institution_configuration_is_created_when_there_is_none_for_a_given_institution()
     {
-        $command                     = new AddRaLocationCommand();
-        $command->raLocationId       = self::uuid();
+        $command                     = new CreateInstitutionConfigurationCommand();
+        $command->institution        = 'An institution';
+
+        $institution                = new Institution($command->institution);
+        $institutionConfigurationId = InstitutionConfigurationId::from($institution);
+
+        $this->scenario
+            ->withAggregateId($institutionConfigurationId)
+            ->when($command)
+            ->then([
+                new NewInstitutionConfigurationCreatedEvent(
+                    $institutionConfigurationId,
+                    $institution
+                )
+            ]);
+    }
+
+    /**
+     * @test
+     * @group command-handler
+     */
+    public function an_institution_configuration_cannot_be_created_when_there_already_is_one_for_a_given_institution()
+    {
+        $this->setExpectedException('Broadway\EventStore\EventStoreException', 'already committed');
```

Comment:
Me too, but I saw FQCN's being used elsewhere (probably because of version constraints back then) so I tried to be consistent.

For the record, it is OK to break this convention as we have bumped our PHP version to 5.6 (which is the required 5.5+).